src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs

```diff
@@ -43,7 +43,9 @@ public string TargetFramework
         }
 
         private List<string> _targetFrameworks = new List<string>();
+        private List<string> _endOfLifeTargetFrameworks = new List<string>();
         public IEnumerable<string> TargetFrameworks { get { return _targetFrameworks; } }
+        public IEnumerable<string> EndOfLifeTargetFrameworks { get { return _endOfLifeTargetFrameworks; } }
```

Comment:
Is this needed for dotnet-svcutil tool?

Reply:
It is used in dotnet-svcutil-lib. I have updated the visibility to internal.